tensorflow/lite/python/lite_test.py

```diff
@@ -481,6 +481,39 @@ def testDefaultRangesStats(self):
     self.assertTrue(([1, 16, 16, 3] == output_details[0]['shape']).all())
     self.assertTrue(output_details[0]['quantization'][0] > 0)  # scale
 
+  def testInvalidStatsUsage(self):
```

Comment:
nit: Name the test `testDefaultRangesStatsInvalid` to follow the pattern of test names.

Reply:
Updated as per the review comments